### ISSUE TITLE
Store variables in external file

### ISSUE THREAD

**Issue by endkb** [endkb/SoundsDownloadScript#1]
Right now, "script-wide" variables are stored at the top of the .ps1. If the script version gets updated, the values need to be transcribed.

Ini file might be a good option, there's already a function in the script to do that. It might need to be moved to a different section.

I don't know how an ini file would work with the remote script blocks.

**Comment by endkb** [comment]
For SoundsDownloadScript, I'm thinking about just allowing dot sourcing a config file from an external .ps1. It's not graceful, but the script blocks for rclone would be a nightmare to convert to another format like xml, json, or ini. I do consider those user configurable items, even though they are more advanced than just changing a simple variable. External file would basically look like the config options in the main script. It would be called with a command line arg.

For genRSS, I might move debug and debugdirectory into the profile files. That only leaves kid3 and rclone locations. Those are easy to set if the script gets updated. genRSS doesn't get updated very often anyway.

**Comment by endkb** [comment]
A downside is that debug logging can't start until the config files are read, so if there's a config file problem, they won't be logged.

Just about everything is in place, I just don't know if I'm ready to pull the trigger on external config files, especially the dot sourcing in the SoundsDownloadScript.

**Comment by endkb** [comment]
Merged into main. Needs testing.

**Comment by endkb** [comment]
I'm satisfied that it works.
